depot/containerstore/containerstore_test.go

```diff
@@ -1765,49 +1775,6 @@ var _ = Describe("Container Store", func() {
 						completeChan = make(chan struct{})
 					})
 
-					testStoppingContainerLogs := func(runErr error) {
-						Context("stopping container logs", func() {
-							var blockSendAppLog chan struct{}
-
-							BeforeEach(func() {
-								megatron.StepsRunnerStub = func(logger lager.Logger, c executor.Container, gc garden.Container, logStreamer log_streamer.LogStreamer, t transformer.Config) (ifrit.Runner, error) {
-									var rf ifrit.RunFunc = func(signals <-chan os.Signal, ready chan<- struct{}) error {
-										close(ready)
-
-										go func() {
-											logStreamer.Stdout().Write([]byte("this should block\n"))
-											logStreamer.Stdout().Write([]byte("bar\n"))
-										}()
-
-										<-completeChan
-										return runErr
-									}
-
-									return rf, nil
-								}
-
-								blockSendAppLog = make(chan struct{})
-								fakeMetronClient.SendAppLogStub = func(message string, sourceType string, tags map[string]string) error {
-									if message == "this should block" {
-										<-blockSendAppLog
-									}
-									return nil
-								}
-							})
-
-							It("ensures logs written after the action exits are not dropped", func() {
```

Comment:
Why was this test case removed?

Reply:
check out the commit message here for an explanation:  https://github.com/cloudfoundry/executor/pull/70/commits/35798d75393c0e8d9a06178b1879e9a6b0cbd2e1

Reply:
👍 thanks!